guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -14,22 +14,19 @@ ifdef::satellite[]
 Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
 endif::[]
 
-ifdef::foreman-el,katello[]
 .Prerequisites
+ifdef::foreman-el,katello[]
 * {Project} {ProjectVersion} running on Enterprise Linux 7.
```

Comment:
This reminds me to revisit https://github.com/theforeman/foreman-documentation/pull/982 so we simply have `{EL}` as a macro.

Reply:
Yes, that would be nice.